src/hide_if_sponsored.js

```diff
@@ -122,6 +120,16 @@ function hideIfSponsored(possibleSponsoredTextQueries, e) {
     return true; // has ad
   }
 
+  if (e.dataset.pagelet === "FeedUnit_1") {
+    let preview = getVisibleText(e);
+    let index = preview.indexOf("·");
+    preview = preview.substring(0, index > 0 ? index : 50).trim() + "...";
+    console.info("ABfF:", `This is the second post and usually is an ad`, [
+      preview,
+      e,
+    ]);
+  }
+
```

Comment:
Did you plan to add this so people can report a new bug easily?

Reply:
added a comment in the code too